src/components/FilterPane/FilterCompose.jsx

```diff
@@ -0,0 +1,39 @@
+import { useFilter } from "../../contexts/FilterContext"
+
+const sortByPrice = ({sortby}, products) => {
+    const sortedList = [...products]
+    switch (sortby){
+        case "HIGH-TO-LOW":
+            return sortedList.sort((prod1, prod2) => prod2.price.discounted - prod1.price.discounted)
+        case "LOW-TO-HIGH":
+            return sortedList.sort((prod1, prod2) => prod1.price.discounted - prod2.price.discounted)
+        default:
+            return sortedList
+    }
+}
+
+const sortByRating = ({rating}, products) => {
+    return products.filter(product => product.rating >= rating)
```

Comment:
it should be filter by instead of sort by

Reply:
Oh right! Thanks for pointing it out.